app/views/replies/_single.haml

```diff
@@ -32,17 +32,25 @@
         .post-author= user_mem_link(reply.user_id, reply.username, reply.user_deleted?)
     - if reply.id.present?
       .post-edit-box
-        = link_to post_or_reply_link(reply), rel: 'alternate'.freeze do
-          = image_tag "icons/link.png".freeze, title: 'Permalink'.freeze, alt: 'Permalink'.freeze
-        - if reply.editable_by?(current_user) || (reply.is_a?(Post) && reply.metadata_editable_by?(current_user))
-          = link_to path_for(reply, 'edit_%s') do
-            = image_tag "icons/pencil.png".freeze, title: 'Edit'.freeze, alt: 'Edit'.freeze
-        - if reply.deletable_by?(current_user)
-          = link_to path_for(reply, '%s'), method: :delete, data: { confirm: "Are you sure you wish to delete this #{reply.class.to_s.downcase}?" } do
-            = image_tag "icons/cross.png".freeze, title: 'Delete'.freeze, alt: 'Delete'.freeze
-        - if reply.is_a?(Reply) && logged_in?
-          = link_to post_path(@post, unread: true, at_id: reply.id), method: :put do
-            = image_tag "icons/eye.png".freeze, title: 'Mark Unread Here'.freeze, alt: 'Mark Unread Here'.freeze
+        - can_be_split = reply.is_a?(Reply) && @post.metadata_editable_by?(current_user) # Logic for splitting a thread at a reply
+        - if params[:split_ui] == 'true' && can_be_split
```

Comment:
This feels like a lot of logic embedded inside here and spread between different parts of the partial.

Can we set a variable up top in posts#show for whether or not the split_ui is enabled (and accessible by the user), then we just rely on that being passed to the partial here for consistency? That way we can make sure the UI enable/disable button, and hiding the reply writer, and adjusting the buttons, is all consistently set up.